MailLibrary/Mail.php

```diff
@@ -248,6 +248,16 @@ public function delete()
 		$this->connection->getDriver()->deleteMail($this->id);
 	}
 
+	public function saveToEml($file)
+	{
+		$this->connection->getDriver()->saveMail($this->id, $file);
+	}
+
+	public function getHeaderInfo()
+	{
+		return $this->connection->getDriver()->getHeaderInfo($this->id);
```

Comment:
You need to switch mailbox before each driver call as IMAP protocol works as access one mailbox per connection at a time.